x-pack/plugins/security_solution/public/management/pages/event_filter/test_utils/index.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { combineReducers, createStore } from 'redux';
+import { Ecs } from '../../../../../common/ecs';
+
+import {
+  MANAGEMENT_STORE_GLOBAL_NAMESPACE,
+  MANAGEMENT_STORE_EVENT_FILTER_NAMESPACE,
+} from '../../../common/constants';
+import { ExceptionListItemSchema } from '../../../../../public/shared_imports';
+
+import { eventFilterPageReducer } from '../store/reducer';
+
+export const createGlobalNoMiddlewareStore = () => {
+  return createStore(
+    combineReducers({
+      [MANAGEMENT_STORE_GLOBAL_NAMESPACE]: combineReducers({
+        [MANAGEMENT_STORE_EVENT_FILTER_NAMESPACE]: eventFilterPageReducer,
+      }),
+    })
+  );
+};
+
+export const event: Ecs = {
```

Comment:
Suggestion: for this one and the ones below, maybe name them so that the word `Mock` is in the name - example: `EcsMock`, `createdEventFilterEntryMock`.  (actually, for this one here, maybe it should be `EcsEventMock`?

also, I woudl suggest converting these to functions that returna  new mock instance everytime in order to prevent/avoid data mutation which could then impact multiple tests. ex: `ecsEventMock = () => {...}`